core/src/autogluon/core/trainer/abstract_trainer.py

```diff
@@ -1134,6 +1134,15 @@ def save(self):
         if self.low_memory:
             self.models = models
 
+    def compile_models(self, **kwargs):
```

Comment:
Add `models` argument so user can specify the models they care about. Default to 'all'.

Add `# TODO: v0.6 Support 'best'`

Reply:
Updated. Now we takes `models` argument, and both `all` and `best` are supported.